src/components/LoginOptions/LoginOptions.stories.mdx

```diff
@@ -0,0 +1,221 @@
+import { Canvas, Meta, Story, ArgsTable } from '@storybook/addon-docs';
+import { userEvent, within, waitFor } from '@storybook/testing-library';
+import { jest, expect } from '@storybook/jest';
+
+import {LiteralsProvider} from 'components/Literal';
+
+import LoginOptions from '.';
+import LoginOptionsDisplay from './LoginOptionsDisplay';
+
+<Meta
+  title="Composites/Login Options"
+/>
+
+# Login Options
+
+The `LoginOptions` component displays the options that are available for a user to authenticate when starting a form.
+
+## Presentation
+
+The `LoginOptionsDisplay` component shows the different buttons that can be clicked to start the authentication process.
+Next to each button, the corresponding logo is displayed.
+
+export const DisplayTemplate = ({loginAsYourselfOptions, loginAsGemachtigdeOptions}) => (
+  <LoginOptionsDisplay
+    loginAsYourselfOptions={loginAsYourselfOptions}
+    loginAsGemachtigdeOptions={loginAsGemachtigdeOptions}
+  />
+);
+
+<Canvas>
+  <Story
+    name="Display"
+    decorators={[(Story) => (
+      <form>
+        {Story()}
```

Comment:
I used `{Story()}` instead of `<Story/>` because I was having this issue: https://github.com/storybookjs/storybook/issues/12596

No idea why it works fine in the language component story.